internal/v2/datasources/apono_access_scopes_datasource.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-log/tflog"
 )
 
-var _ datasource.DataSource = &AponoAccessScopesDataSource{}
+var _ datasource.DataSourceWithConfigure = &AponoAccessScopesDataSource{}
```

Comment:
this is more correct. Configure is required. So now I'm enforcing it in all resources and datasources.